docs/conf.py

```diff
@@ -70,9 +70,11 @@
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
-    'packaging': ('https://packaging.python.org/en/latest/', None),
+    'packaging': ('https://packaging.python.org/', None),
```

Comment:
This name would be more precise.
```suggestion
    'PyPUG': ('https://packaging.python.org', None),
```

Reply:
Looking closer, I think the correct fix would be to apply
```suggestion
    'packaging': ('https://packaging.pypa.io/en/latest', None),
```
(and maybe drop the ignore)